questdb-rs/src/ingress/mod.rs

```diff
@@ -2364,6 +2372,126 @@ impl SenderBuilder {
     }
 }
 
+pub struct TlsConnector {
```

Comment:
Move to `http`, or it will be compiled in even if HTTP is disabled in the feature flags.